server/src/controllers/folders.controller.ts

```diff
@@ -27,9 +27,9 @@ export class FoldersController {
     if (!ability.can('createFolder', Folder)) {
       throw new ForbiddenException('You do not have permissions to perform this action');
     }
-    const folderName = req.body.name;
+    createFolderDto.name = req.body.name;
```

Comment:
Why should we need to do this? `name` will be already available in `createFolderDto`

Reply:
I think I get you for both these comments! Thank you! Still trying to get my head round DTOs